packages/@aws-cdk/aws-rds/lib/instance-engine.ts

```diff
@@ -258,6 +258,10 @@ export class MariaDbEngineVersion {
   public static readonly VER_10_2_40 = MariaDbEngineVersion.of('10.2.40', '10.2');
   /** Version "10.2.41". */
   public static readonly VER_10_2_41 = MariaDbEngineVersion.of('10.2.41', '10.2');
+  /** Version "10.2.43". */
+  public static readonly VER_10_2_43 = MariaDbEngineVersion.of('10.2.43', '10.2');
+  /** Version "10.2.44". */
+  public static readonly VER_10_2_44 = MariaDbEngineVersion.of('10.2.44', '10.2');
```

Comment:
According to the [MariaDB on Amazon RDS versions](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/MariaDB.Concepts.VersionMgmt.html) page, you can no longer create a new 10.2 instance as of July 15 so I'm not sure we should be adding these here.

Reply:
Fair enough, I missed that note. Removed as you suggested!